src/__tests__/index.test.ts

```diff
@@ -177,3 +177,115 @@ describe('send', () => {
     expect(console.log).toHaveBeenCalled();
   });
 });
+
+describe('retract', () => {
+  const retractEndPointRegex = /.*\/sender\/retract/;
+  const notificationId = 'notificationId';
+  const userId = 'userId';
+  const secondaryId = 'secondaryId';
+  const clientId = 'testClientId';
+  const clientSecret = 'testClientSecret';
+
+  test('Returns a Promise<AxiosResponse>', async () => {
```

Comment:
There are some similar tests with 'send'.
Please take a look at following link and make tests more generic if it is possible. 
https://jestjs.io/docs/api#testeachtablename-fn-timeout